heat/core/tests/test_statistics.py

```diff
@@ -301,14 +301,6 @@ def test_mean(self):
                             self.assertEqual(res.split, len(target_dims) - 1)
                         else:
                             self.assertEqual(res.split, z.split)
-
```

Comment:
Please reorder mean(), std() and var() to appear in alphabetical order in their respective files including test cases.